velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -376,7 +407,16 @@ void AggregationFuzzer::go() {
             (signature.name.find("approx_") == std::string::npos) &&
             vectorFuzzer_.coinToss(0.2);
 
-        auto call = makeFunctionCall(signature.name, argNames, sortedInputs);
+        // Exclude approx_xxx aggregations since their semantics may conflict
```

Comment:
This explanation is a bit strange. Results of f(distinct x) are different from f(x) for many aggregations, not only approx_. For example, array_agg(distinct x) != array_agg(x).